apps/src/applab/applab.js

```diff
@@ -1229,6 +1240,28 @@ function onInterfaceModeChange(mode) {
 }
 
 /**
+ * Display the workspace corresponding to the specified mode.
+ * @param {ApplabInterfaceMode} mode
+ */
+Applab.setWorkspaceMode = function (mode) {
+  var enable = (ApplabInterfaceMode.DESIGN === mode);
+  var designWorkspace = document.getElementById('designWorkspace');
+  // TODO(dave): remove unneeded designWorkspace existence check
```

Comment:
Nope... I initially put it off for fear that it might break something, but I did mean to address it before checkin. Any cases to consider besides a level with `config.level.hideDesignMode`?